tests/foreman/ui_airgun/test_dashboard.py

```diff
@@ -70,11 +73,16 @@ def test_positive_host_configuration_status(session):
         ' status.failed_restarts > 0) and status.enabled = true',
         'last_report > \"30 minutes ago\" and status.enabled = true and'
         ' status.applied = 0 and status.failed = 0 and status.pending = 0',
-        'status.pending > 0 and status.enabled = true',
+        'last_report > \"30 minutes ago\" and status.pending > 0'
+        ' and status.enabled = true',
         'last_report < \"30 minutes ago\" and status.enabled = true',
+        'last_report > \"30 minutes ago\" and status.enabled = false',
         'not has last_report and status.enabled = true',
-        'status.enabled = false'
     ]
+    if bz_bug_is_open(1631219):
+        criteria_list.pop()
```

Comment:
Maybe it's worth to explicitly specify what value to pop() as otherwise we shall remember not to change order of items in both lists? Not a blocker as I don't have strong opinion about this

Reply:
I move them to last position, to not to have to use that values, the lists are in same list and following the logic flow so think no worries here.